src/containers/Profile/ProfileDetails.js

```diff
@@ -22,6 +24,46 @@ import { ipfsToHttp } from 'utils/helpers';
 import { profileUISelector } from './selectors';
 import { actions } from './reducer';
 
+const isEmptyProperty = property => {
+  return (
+    property === undefined ||
+    property === null ||
+    (typeof property === 'object' && Object.keys(property).length === 0) ||
+    (typeof property === 'string' && property.trim().length === 0)
+  );
+};
+
+const getProfileCompletePercent = user => {
+  const userElsewhereInfo = {
+    github: user.github,
+    linkedin: user.linkedin,
+    twitter: user.twitter,
+    website: user.website
+  };
+
+  const userElsewhereValues = Object.values(userElsewhereInfo).filter(
+    value => value
+  ).length;
+
+  const userMinRequiredInfo = {
+    name: user.name,
+    organization: user.organization,
+    languages: user.languages,
+    skills: user.skills,
+    email: user.email,
+    elsewhere: userElsewhereValues > 0 ? userElsewhereValues : undefined
+  };
+
+  const keys = Object.keys(userMinRequiredInfo).length;
+  const nonEmptyValues = Object.values(userMinRequiredInfo).filter(
```

Comment:
here you'll want to import `pickBy` from lodash and do `pickBy((key, value) => !isEmptyProperty(key), userMin)`